<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget { background-color: white; }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="infoTab">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>261</width>
      <height>281</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Info</string>
     </attribute>
     <widget class="QLabel" name="img_info">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>250</width>
        <height>250</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Channel</string>
     </attribute>
     <widget class="QLabel" name="plotLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>250</width>
        <height>250</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>250</height>
       </size>
      </property>
      <property name="text">
       <string>Please select channel to extract</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="image_view">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>700</width>
      <height>500</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDockWidget" name="bnwDock">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>261</width>
      <height>141</height>
     </rect>
    </property>
    <property name="floating">
     <bool>true</bool>
    </property>
    <property name="allowedAreas">
     <set>Qt::RightDockWidgetArea</set>
    </property>
    <property name="windowTitle">
     <string>Black and White Threshold</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_4">
     <widget class="QSlider" name="bnwSlider">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>127</number>
      </property>
      <property name="sliderPosition">
       <number>127</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="bnwVal">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>70</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Value = 127</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="saveImage">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QDockWidget" name="gammaDock">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>350</y>
      <width>261</width>
      <height>141</height>
     </rect>
    </property>
    <property name="floating">
     <bool>true</bool>
    </property>
    <property name="allowedAreas">
     <set>Qt::RightDockWidgetArea</set>
    </property>
    <property name="windowTitle">
     <string>Gamma Threshold</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_2">
     <widget class="QSlider" name="gammaSlider">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="sliderPosition">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="gammaVal">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>70</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Gamma = 1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="saveImage_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Image"/>
   <addaction name="grayscaleTool"/>
   <addaction name="tfNegative"/>
   <addaction name="reset"/>
   <addaction name="spatial_filtering"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuChannel">
    <property name="title">
     <string>Channel</string>
    </property>
    <addaction name="getRed"/>
    <addaction name="getGreen"/>
    <addaction name="getBlue"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="reset"/>
   </widget>
   <widget class="QMenu" name="menuTransform">
    <property name="title">
     <string>Transform</string>
    </property>
    <addaction name="tfGrayscale"/>
    <addaction name="tfNegative"/>
    <addaction name="thGamma"/>
    <addaction name="bnwButton"/>
   </widget>
   <widget class="QMenu" name="menuSpatial_Filters">
    <property name="title">
     <string>Spatial Filters</string>
    </property>
    <addaction name="open_filters"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuChannel"/>
   <addaction name="menuTransform"/>
   <addaction name="menuSpatial_Filters"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_Image">
   <property name="icon">
    <iconset>
     <normaloff>icons/plus.png</normaloff>icons/plus.png</iconset>
   </property>
   <property name="text">
    <string>Add Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="showChannels_2">
   <property name="icon">
    <iconset>
     <normaloff>../../../Downloads/351428.png</normaloff>../../../Downloads/351428.png</iconset>
   </property>
   <property name="text">
    <string>splitChannel</string>
   </property>
  </action>
  <action name="getRed">
   <property name="text">
    <string>Extract Red</string>
   </property>
  </action>
  <action name="getBlue">
   <property name="text">
    <string>Extract Blue</string>
   </property>
  </action>
  <action name="getGreen">
   <property name="text">
    <string>Extract Green</string>
   </property>
  </action>
  <action name="showChannels">
   <property name="text">
    <string>Show Channels</string>
   </property>
  </action>
  <action name="tfGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="tfNegative">
   <property name="icon">
    <iconset>
     <normaloff>icons/minus.png</normaloff>icons/minus.png</iconset>
   </property>
   <property name="text">
    <string>Negative</string>
   </property>
  </action>
  <action name="thGamma">
   <property name="text">
    <string>Gamma</string>
   </property>
  </action>
  <action name="bnwButton">
   <property name="text">
    <string>Black and White</string>
   </property>
  </action>
  <action name="reset">
   <property name="icon">
    <iconset>
     <normaloff>icons/undo-arrow.png</normaloff>icons/undo-arrow.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="grayscaleTool">
   <property name="icon">
    <iconset>
     <normaloff>icons/black-and-white.png</normaloff>icons/black-and-white.png</iconset>
   </property>
   <property name="text">
    <string>GRAYSCALE</string>
   </property>
  </action>
  <action name="open_filters">
   <property name="text">
    <string>Open Filters</string>
   </property>
  </action>
  <action name="mdn_filter">
   <property name="text">
    <string>Median Filter</string>
   </property>
  </action>
  <action name="hpass_filter">
   <property name="text">
    <string>Highpass Filter</string>
   </property>
  </action>
  <action name="unsharp_filter">
   <property name="text">
    <string>Unsharp Masking</string>
   </property>
  </action>
  <action name="hboost_filter">
   <property name="text">
    <string>Highboost Filtering</string>
   </property>
  </action>
  <action name="sobel_gradient">
   <property name="text">
    <string>Sobel Gradient</string>
   </property>
  </action>
  <action name="spatial_filtering">
   <property name="icon">
    <iconset>
     <normaloff>icons/filter-filled-tool-symbol.png</normaloff>icons/filter-filled-tool-symbol.png</iconset>
   </property>
   <property name="text">
    <string>Spatial Filtering</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
